.travis.yml

```diff
@@ -5,5 +5,7 @@ node_js:
   - "4"
   - "5"
   - "6"
+env:
+  - TZ="Europe/Copenhagen"
```

Comment:
I'm -1 for that. Since a `Date` object is always in local time, I think we have a responsibility of maintaining that in the output. Maybe a better solution is to move that `TZ="Europe/Copenhagen"` into the `npm test` script as a bootstrapping mechanism, which is actually [the same thing we do for the `moment`-related tests that depend on the timezone being `Europe/Copenhagen`](https://github.com/unexpectedjs/unexpected-moment/blob/master/bootstrap-unexpected-markdown.js#L7).